### PR TITLE
Feature/nres fields

### PR DESCRIPTION
**Pull request by jnation3406**
Rob said hes using this stuff, so we do need it for intiial deploy

### CODE REVIEW

**Comment by Fingel** [comment]
Adding @ire-and-curses  as we discussed not wanting to blow up our request language with unnecessary fields

**Comment by jnation3406** [comment]
I love when you add Ire and Curses to my pull request

**Comment by Fingel** [comment]
Seems fine to me to merge this. However the PR is pretty sparse. As it stands with this commit NRES will work exactly like FLOYDS. Now we have a bunch of optional parameters that can be passed through to any request. None of them appear to be required. Any business logic that applies to FLOYDS (ex. slit is required) will apply to NRES as well. Perhaps we don't care at the moment, but this cannot be all there is to do.

**Comment by ire-and-curses** [comment]
Well let's talk about that then. NRES doesn't have a slit (for example) so we need to be careful about the validation logic we apply here.

**Comment by jnation3406** [comment]
Yeah if its requiring a slit for the nres observation types then I must have missed something.

**Comment by Fingel** [comment]
Yes, we have to be careful about this branch: 
https://github.com/LCOGT/valhalla/blob/master/valhalla/userrequests/serializers.py#L76
It also means ag_mode will default to ON, acquire_mode will default to WCS, etc.

**Comment by Fingel** [comment]
This is where slits are required: https://github.com/LCOGT/valhalla/blob/master/valhalla/userrequests/serializers.py#L89

The logic here is a bit opaque.

**Comment by Fingel** [comment]
Wait, so NRES exposures aren't SPECTRUMs?

**Comment by jnation3406** [comment]
NRES exposures are not SPECTRUMs. It is all as he who shall not be named wishes. Everyday, as we creep ever closer to our impen̡d͡i҉n͘g͘ ̸ en̷d҉, the sins of the NRES engin͝e͟er̸ become the sins of the s͘of͞tw͘arè te҉a͡m̵ and a new mo̡ĺe̢ćul͟e ̛t͡ype is born. L͜o̷ng ̡ĺive ̨h͞e͞-w͡h̕o͟-̷sha̷ll͜-͝no͜t-b͘e-na̢m҉e̶d̨!̶ ͟L͜ong ̧l҉ive͞ ́t́he̢ ͜co͜n҉ćurr҉e͏ncy k̴in͢g̡ A̭͔̯͈H̻̙̤̜͙̪̞͠h͕̣̳̣̫h̶͖͉h͇̯͘ḩ̲̮h̞͘hhh̯h҉͔̦͚͓̰ḩ̻hh͙̻͈h͘h̺̫͖̟͙̲͎h̴̯h̵̪̲̻̯h̻͖̼̦̫h͖͟ͅh̢̳̝̪̱h̲̬h̵̫͍̠h͞ḥ̘͔̯̫͍͑ͮͤ̇̉̐ḣ̼̹̟̆ͮ͆̿͑ͣͅh̟̙̩̗͉͙̼̍ͮh̐ͤͥĥ̙̼͈̞͓̹͔͆̓̂͒͋h̝͖̱̅̈͌͂̌h͕̜̳ͪͯ̒͆̉ͮh͇̺̙͚̘̠͉ͤ̓̂̆̏h̊ͪ͊̈́͒h̥̯͓̑̀̃͛h̖̭͎̜̱͎̬̓̔̾̆̅͋̂h̹̠̺͊̈̂̅ͅh̙̻̉ͯͧ̎̏ͫ͒h̰͚̟͕̤̰ͅh̝͢ẖ̡͙͈̠h̬̻̼͇̭̳͜h͕̥̻͓h̜̬̘͔͚̱̗͡h̵̪̳̖͎̠̟̖hh̹͝h̩̹̙̪̞̫̰h̕h̙̩h̢̲h͙͉̘̗̰̗͉͠h̹̞͕̳̹

**Comment by Fingel** [comment]
....
Forget it. I'll wait until Monday to deal with this ₴ℌⅈ⊤

File: valhalla/userrequests/target_helpers.py [hunk 1]
```diff
@@ -42,8 +42,8 @@ def data(self):
 class SiderealTargetHelper(BaseTargetHelper):
     def __init__(self, target):
         self.fields = (
-            'type', 'name', 'ra', 'dec', 'proper_motion_ra', 'proper_motion_dec', 'parallax',
-            'coordinate_system', 'equinox', 'epoch', 'acquire_mode', 'rot_mode', 'rot_angle'
+            'type', 'name', 'ra', 'dec', 'proper_motion_ra', 'proper_motion_dec', 'parallax', 'radvel',
+            'coordinate_system', 'equinox', 'epoch', 'acquire_mode', 'rot_mode', 'rot_angle', 'vmag'
```
Comment (Fingel) [inline valhalla/userrequests/target_helpers.py]: I'm pretty sure NonSidereal targets have a magnitude as well. In fact all targets do.

**Pull request merged**